### DISCUSSION TITLE
Madal not respond to KeyboardAvoidingView

### DISCUSSION THREAD

**Discussion by PawelGIX** [srivastavaanurag79/react-native-paper-select#51] [Ideas]
Modal window is not keyboard aware. 
I'm using <KeyboardAvoidingView>
However, the modal does not seem to respond to it.
![simulator_screenshot_31098485-1C7C-41BB-AC1B-AA6CB419C5EC](https://github.com/srivastavaanurag79/react-native-paper-select/assets/391153/acee4470-52aa-45dc-b9ac-aadd72344c75)
<img width="469" alt="image" src="https://github.com/srivastavaanurag79/react-native-paper-select/assets/391153/8fd76e46-7e02-4beb-aba5-040e01a7114d">

**Comment by srivastavaanurag79** [comment]
will let you know when fixed

**Comment by srivastavaanurag79** [comment]
To make the React Native Paper Modal window keyboard-aware, you can use the react-native-keyboard-aware-scroll-view library in combination with the KeyboardAvoidingView

```
<KeyboardAwareScrollView
            contentContainerStyle={{ flex: 1 }}
            extraScrollHeight={Platform.select({ ios: 20, android: 20 })}
            enableOnAndroid
          >
```
